README.md

```diff
@@ -27,8 +27,30 @@ Anime2Sketch is a sketch extractor that works well on illustration, anime art, a
 
 
 ## Get Started
-### Installation 
-Install the required packages: ```pip install -r requirements.txt```
+### Installation using Conda Environment
+```
+##Note: tested on Ubuntu 18.04 using Cuda 10.2
+git clone https://github.com/Mukosame/Anime2Sketch.git
+cd ./Anime2Sketch
+conda env create -f environment.yml
+conda activate a2s
+```
+
+### Installation using Conda commands
+```
+##Note: You can change the Cuda version to the one that you desire, here it is 10.2
+git clone https://github.com/Mukosame/Anime2Sketch.git
+cd ./Anime2Sketch
+conda create -n a2s python=3.6
+conda activate a2s
+conda install pytorch torchvision torchaudio torchtext cudatoolkit=10.2 -c pytorch
```

Comment:
It doesn't need torchaudio or torchtext. 
Besides, now AK31 adds gradio to the dependency list.
I still hope to keep the whole project and dependencies as simple as possible to make it easier to run on any environments :)